odxtools/parameterinfo.py

```diff
@@ -20,12 +26,65 @@
 from .parameters.parameter import Parameter
 from .parameters.parameterwithdop import ParameterWithDOP
 from .parameters.reservedparameter import ReservedParameter
+from .parameters.systemparameter import SystemParameter
 from .parameters.tablekeyparameter import TableKeyParameter
 from .parameters.tablestructparameter import TableStructParameter
 from .paramlengthinfotype import ParamLengthInfoType
 from .staticfield import StaticField
 
 
+def _get_type_info(odx_type: DataType) -> str:
```

Comment:
I have a feeling this should be in the __str__ or __repr__ function of DataType

Reply:
certainly not `__repr__()`, because if I got this correctly `__repr__()` returns a string containing all the information required to re-create the respective object. for `__str__()`, this is a matter of taste, but I think that for most contexts where a `DataType` object shall be printed, this is a bit too little information?

Reply:
It's enough for 99% of use, the only thing that's a bit hidden here is the string encoding, and most ODX files use UTF-8 or ASCII, so encoding does not really matter much for them.

Reply:
ok, for `DataType`, I budge ;): https://github.com/mercedes-benz/odxtools/pull/348/commits/6503e1c17fa74667e407144dfead4d6069a05d9e (for the segment objects, I still think it makes little sense but it would constitute a (minor) footgun...)